samples/training/train-on-amlcompute/train-on-amlcompute.R

```diff
@@ -29,7 +29,7 @@ est <- estimator(source_directory = ".",
                  entry_script = "train.R",
                  script_params = list("--data_folder" = ds$path(target_path)),
                  compute_target = compute_target,
-                 cran_packages = c("caret", "optparse", "e1071"))
+                 cran_packages = c("caret"))
```

Comment:
caret is also not needed right? It is part of r-essentials bundle I believe

Reply:
you're right. thanks for pointing it out. will remove it as well